app/javascript/react/assets/styles/Typography/Typography.style.tsx

```diff
@@ -37,6 +49,7 @@ const H6 = styled.h6<{ bold?: boolean; uppercase?: boolean }>`
   font-size: 10px;
   text-transform: ${(p) => (p.uppercase ? "uppercase" : "none")};
   letterspacing: ${(p) => (p.uppercase ? "0.1px" : "normal")};
+  color: inherit
```

Comment:
We can add a default color to the main tag like <body>, so all the text as a default will be the same and this inherit should not be needed.

Reply:
As well as Roboto font-family - should work.

Reply:
Once it is fixed 😅

Reply:
What do you mean by _to the main tag like_? How can it be done efficiently?